05/app.js

```diff
@@ -1,53 +1,77 @@
 const stats = {
-    paragraphs: {
-        'p1': 0,
-    },
-    links: {
-        '/dolor.html': 0,
-    }
+	paragraphs: {
+		p1: 0,
+	},
+	links: {
+		'/dolor.html': 0,
+	},
 };
 
 /* tutaj umieść swój kod */
 
+let counterText = 0;
+const paragraphsArr = document.querySelectorAll('.text');
+paragraphsArr.forEach((el) => {
+	// console.log(el);
+	el.addEventListener('click', () => {
+		counterText++;
+		stats.paragraphs['p1'] = counterText;
+	});
+});
+
+let counterLinks = 0;
+const linksArr = document.querySelectorAll('a[href].link');
+linksArr.forEach((el) => {
+	if (el.getAttribute('href') == Object.keys(stats.links)) {
+		el.addEventListener('click', (e) => {
+			e.preventDefault();
+			counterLinks++;
+			stats.links['/dolor.html'] = counterLinks;
+		});
+	}
+});
```

Comment:
To zadanko jest do poprawy. Klikanie w inny link niż "dolor" powoduje przekierowanie.
Kliknięcie w inny paragraf niż pierwszy nie powoduje aktualizacji dla `p2` lub `p3` tylko `p1`